server/asset/eth/common.go

```diff
@@ -81,19 +81,21 @@ const (
 // to the smart contract. This type of ID is useful to identify coins that
 // were sent in transactions that have not yet been mined.
 type TxCoinID struct {
-	TxID [32]byte
+	TxID  [32]byte
+	Index uint32
```

Comment:
Will you please document Index in the TxCoinID docs when you rebase.

Reply:
Done

Reply:
Yeesh I'm blind, it was already described.  Thanks

Reply:
No it wasn't haha, I did one update as a part of the rebase, then I did another small update.